Homework1/exam14.py

```diff
@@ -0,0 +1,21 @@
+# 14 lines: Doctest-based testing
+import doctest
+
```

Comment:
2 lines between import and code. (1 in other cases). it should be consistent everywhere